tests/unit/test_cli.py

```diff
@@ -0,0 +1,31 @@
+from unittest.mock import Mock, patch
+
+from prometheus_juju_backup_all_exporter.__main__ import main, parse_command_line
+
+
+class TestCli:
+    """Cli test class."""
+
+    @patch("argparse.ArgumentParser")
+    def test_parse_argument(self, mock_argument_parser):
+        parse_command_line()
+        mock_argument_parser.assert_called_once()
+
+    @patch("prometheus_juju_backup_all_exporter.__main__.parse_command_line")
+    @patch("prometheus_juju_backup_all_exporter.__main__.Exporter")
+    @patch("prometheus_juju_backup_all_exporter.__main__.Config")
+    @patch("logging.getLevelName")
```

Comment:
(non-blocker), just a hint on how to avoid verbose mocking. If you'd import
```python
from prometheus_juju_backup_all_exporter import __main__
```
you could patch like this:
```python
@patch.object(__main__, "Config")
...
```

Reply:
Thanks, that's a nice suggestion